src/views/CryptoChart/index.jsx

```diff
@@ -1,20 +0,0 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { AppContainer } from 'react-hot-loader';
-import CryptoChart from './CryptoChart';
-import './styles.css';
-
-const render = (Component) => {
-  ReactDOM.render(
-    <AppContainer>
-      <Component />
-    </AppContainer>,
-    document.getElementById('root')
-  );
-};
-
-render(CryptoChart);
-
-module.hot.accept('./CryptoChart', () => {
-  render(CryptoChart);
-});
```

Comment:
couldn't tell if this was used, appeared to be an artifact but I kept getting dropped in their when searching for the other crypto chart so I deleted just to stop getting confused :)